mdpow/__init__.py

```diff
@@ -2,13 +2,9 @@
 # Copyright (c) 2010 Oliver Beckstein <orbeckst@gmail.com>
 # Released under the GNU Public License 3 (or higher, your choice)
 # See the file COPYING for details.
+import mdpow.log as log
 
-
-from __future__ import absolute_import
-
-from . import log
-
-from ._version import get_versions
+from mdpow._version import get_versions
```

Comment:
The import from . should remain, that's the correct way of ensuring that we get exactly that `._version`